phase1/config.ini.example

```diff
@@ -17,6 +17,10 @@ channel = #example-channel
 nickname = example-builder
 password = example
 
+#[webhook]
```

Comment:
Can we make this future proof and allow more hooks from the start, thus handling it as branches/workers?

Reply:
Done with the reimplementation.